impeller/display_list/display_list_dispatcher.cc

```diff
@@ -762,7 +762,38 @@ void DisplayListDispatcher::drawArc(const SkRect& oval_bounds,
 void DisplayListDispatcher::drawPoints(SkCanvas::PointMode mode,
                                        uint32_t count,
                                        const SkPoint points[]) {
-  UNIMPLEMENTED;
+  // auto path = PathBuilder{}.AddLine(ToPoint(p0), ToPoint(p1)).TakePath();
+  Paint paint = paint_;
+  paint.style = Paint::Style::kStroke;
+  switch (mode) {
+    case SkCanvas::kPoints_PointMode:
+      if (paint.stroke_cap == SolidStrokeContents::Cap::kButt) {
+        paint.stroke_cap = SolidStrokeContents::Cap::kSquare;
+      }
+      for (uint32_t i = 0; i < count; i++) {
+        SkPoint p0 = points[i];
+        SkPoint p1 = points[i] + SkPoint{0.0001, 0.0};
```

Comment:
Maybe use `kEhCloseEnough` for this hack? 🤣

Reply:
Done. See https://github.com/flutter/flutter/issues/109077 and the new comment about `kEhCloseEnough`